web/ui/two-steps-page.svelte

```diff
@@ -23,18 +24,26 @@
   }
 
   onMount(() => {
+    window.addEventListener('keydown', toggleSide)
     document.title = title + ' › ' + prevTitle
     return () => {
+      window.removeEventListener('keydown', toggleSide)
       document.title = prevTitle
     }
   })
 </script>
 
 <main id="page" class="two-steps-page">
-  <div bind:this={first} class="two-steps-page_step">
+  <div
+    bind:this={first}
+    class={`two-steps-page_step ${$side === 'second' ? 'two-steps-page_step-desktop-only' : ''}`}
```

Comment:
In BEM, you can’t put 2 elements on the same tag.

I think `two-steps-page_step-desktop-only` should be a modifier like `is-*`.

And the current name is not clear for me. Is it `is-current`?